python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -1142,6 +1160,31 @@ def submit_run_request(
                     )
                     updated_evaluation_keys.add(entity_key)
 
+            # check if the sensor is still enabled:
+            if num_submitted >= batch_size:
+                num_submitted = 0
```

Comment:
if num_submitted % batch_size == 0?

Reply:
sure - is there a reason that doing the mod is better? because it lets us not reset the counter to 0?

Reply:
yeah, i think they're equivalent - it just read a little weird to see num_submitted = 0 when more than 0 runs have been submitted